go.mod

```diff
@@ -1,6 +1,8 @@
 module istio.io/istio
 
-go 1.21
+//go:built go 1.22
+
+toolchain go1.22.3
```

Comment:
The upstream 1.20 is built by go 1.21 :
https://github.com/istio/istio/blob/release-1.20/go.mod

I don't think we should upgrade it here to 1.22.3

Reply:
I changed it back to 1.21